sonic-thermalctld/scripts/thermalctld

```diff
@@ -752,6 +756,23 @@ class ThermalMonitor(ProcessTaskBase):
 
         self.logger.log_info("Stop thermal monitoring loop")
 
+    def signal_handler(self, sig, frame):
```

Comment:
the signal_handler is not relevant because it is not hocked.

Reply:
not understand the comment here, could you elaborate?

Reply:
I mean this function was committed by mistake and should be removed.
I just left a comment to remind myself.